app/views/users/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>User Profile</h1>
+<p>Name: <%=current_user.user_detail.name%></p>
+<p>Email: <%=current_user.email_address%></p>
+<p>Name: <%=current_user.user_detail.street_address%></p>
+<p>Name: <%=current_user.user_detail.state%></p>
+<p>Name: <%=current_user.user_detail.city%></p>
+<p>Name: <%=current_user.user_detail.zip_code%></p>
+
```

Comment:
Please change the labels to match the data

Reply:
Great Catch

Reply:
requested changes have been made